devtools/maven/src/main/java/io/quarkus/maven/ListExtensionsMojo.java

```diff
@@ -16,7 +16,7 @@
  * List the available extensions.
  * You can add one or several extensions in one go, with the 2 following mojos:
  * {@code add-extensions} and {@code add-extension}.
- * You can list all extension or just installable and choose simple or full format.
+ * You can list all extension or just installable and choose name, simple or full format.
```

Comment:
choose between 3 output format: `name`, `simple` and `full`. (can you add a few words for each)